### ISSUE TITLE
Code for Miami -> Code for South?

### ISSUE THREAD

**Issue by themightychris** [codeforamerica/brigade-information#126]
I noticed https://github.com/Code-for-Miami 404ing and it looks like the repos redirect here now: https://github.com/CodeforSouth/

Does the whole Code for Miami entry need redoing?

**Comment by tdooner** [comment]
Yeah, they're in the midst of a rename to Code for South [Florida]. I'm not sure what the grand plan is for it so let me find out and get back to you on this.
